setup/so-whiptail

```diff
@@ -33,6 +33,24 @@ whiptail_airgap() {
 	INTERWEBS=$(echo "${INTERWEBS^^}" | tr -d ' ')
 }
 
+whiptail_analyst_install() {
+
+	[ -n "$TESTING" ] && return
+
+	read -r -d '' message <<- EOM
+	
+	Welcome to the Security Onion Analyst Workstation install!
+
+	You can choose to join this workstation to an existing grid?
```

Comment:
Let's change this to:

Would you like to join this workstation to an existing grid?